### ISSUE TITLE
Use less div in the base theme

### ISSUE THREAD

**Issue by facelesspanda** [MoOx/phenomic#795]
I've been looking at the html the default theme produced on its homepage.
And I saw `<div>` under `<div>` under `<div>`, and I gotta ask myself:

>  Why `<div>`s?

There are a lot of html tags out there. The theme makes use of `<header>` and `<footer>`, but why doesn't it make use of tags like `<article>` or `<section>` ?

I get that there's gonna be a LOT of nesting, when you deal with react components, but why always a div?

For example, the BodyContainer wraps in a `<div>`. Why ? Why not a `<section>` since it's what it'll end up being, on the layout : the page body section.
Or if you don't like that you could have an `<article>` in the BodyContainer and a `<section>` in the Page layout, no? Or even the PageContainer?

Why `<div>` everywhere?

---

NB: I'm gonna make the changes I feel I need for my projects anyway, this is not a complain, it's a simple curiosity-based question: why `<div>` ?

**Comment by MoOx** [comment]
0.17.0 remove some unnecessary div.
Why are article & section better?

But yeah for some remaining div, that can be an improvement.

**Comment by facelesspanda** [comment]
There not better in that performance is improved, but to me a markup tree should be semantically structured. Div's should be used when nothing else can be used, in my opinion.

**Comment by facelesspanda** [comment]
There not better in that performance is improved, but to me a markup tree should be semantically structured. Div's should be used when nothing else can be used, in my opinion.

**Comment by lionelB** [comment]
I think it could be a good idea to follow known best practice for markup and use tags like`<nav>, <header><article>, <aside>, <section> ...` in the default template

**Comment by thangngoc89** [comment]
Semantic tag is better. Yes. But something with React, it's hard to follow semantic because of its component nature. If you can send a PR, I'll be happy to accept the change to base theme. @facelesspanda

**Comment by MoOx** [comment]
Guys, I tagged this issue to "contribution welcome" since the beginning ;)

**Comment by MoOx** [comment]
Will be a thing in v1 example.
